src/App.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from "react";
+import Todo from "./Components/Todo";
+import TodoForm from "./Components/TodoForm";
+
+import logo from "./logo.svg";
+import "./App.css";
+
+function App() {
+    const initialTodo = [
+        {
+            id: 1,
+            task: "Learn ReactJs",
+            completed: false
+        },
+        {
+            id: 2,
+            task: "Learn VueJs",
+            completed: false
+        }
+    ];
+    const [todoList, updateTodoList] = useState(initialTodo);
+
+    const handleCompleted = id => {
+        let newTodoList = [...todoList];
+
+        newTodoList.map(todo => {
+            if (todo.id === id) {
+                todo.completed = true;
+                return todo;
+            }
+        });
+        updateTodoList(newTodoList);
+    };
+
+    const handleAdd = todo => {
+        let newTodoList = [...todoList];
+        newTodoList.push(todo);
+        updateTodoList(newTodoList);
+    };
+
+    return (
+        <div className="App">
+            <div className="App-header">
+                <h1> MY TODO LIST </h1>
+                <div className="frame">
+                    <div className="container">
+                        <ul className="list-group">
+                            <Todo
```

Comment:
If you think about why Components are so good for websites? Reusability! So really you wanna have `Todo` as being one list item, that is reused how ever many times long the list is.

Could be something like;
```
<TodoList>
    <TodoItem text="Learn ReactJS" onClick={handleClick} />
    <TodoItem text="Learn VueJS" onClick={handleClick} />
    <TodoItem text="I wouldn't bother with AngularJS" onClick={handleClick} />
</TodoList>
```
Or better yet;
```
const handleClick = () => console.log("I am clicked")
<TodoList>
    {todoListState.map(todoItemObject => {
        <TodoItem text={todoItemObject.text} onClick={handleClick} />
    })}
</TodoList>
```
Hopefully this makes it a bit more clearer in what I mean.